pyht/client.py

```diff
@@ -43,20 +45,96 @@
     ]
 
 
+class HTTPFormat(Enum):
```

Comment:
Don't we support "RAW" where we don't encode ?

Reply:
FAL API does not support RAW

Reply:
I wrote to them to add it. I will make sure they do or I will do it.

Reply:
https://github.com/fal-ai-community/playht-tts/issues/48

Reply:
Got it, will add